tests/postgres_test.py

```diff
@@ -6,7 +6,7 @@
 def test_generate_connection():
     from dbt_sugar.core.connectors.postgres_connector import PostgresConnector
 
-    conn = PostgresConnector(**CREDENTIALS).generate_connection()
+    conn = PostgresConnector(**CREDENTIALS)
```

Comment:
we make is so that when we init we always create a connection?

Reply:
yes, well is not a connection is the engine for making later the connection.

Reply:
Got it! Sounds good anyway, just wanted to make sure.